### DISCUSSION TITLE
loongarch64打包请求

### DISCUSSION THREAD

**Discussion by donmor** [natfrp/wiki#345] [想法] [answered]
Go1.19加入了完整的loongarch64支持，在不使用cgo时，可以直接配置"GOOS=linux GOARCH=loong64"进行交叉编译。loongarch64是龙芯于3*5000系列引入的新架构。

[a comment by an automated account is not shown]

**Reply by donmor** [reply]
已确认工作正常。后续会加入自动构建并发布吗？

**Reply by donmor** [reply]
感谢！

**Reply by fakeboboliu** [reply]
docker 自动构建现在也会提供 `linux/loong64` 版本。但是 runc 目前还没有支持长儿，就当先准备着了（

[a comment by an automated account is not shown]
